docs/content/querying/sorting-orders.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc_page
+---
+
+# Sorting Orders
+
+These sorting orders are used by the [TopNMetricSpec](./topnmetricspec.html), [SearchQuery](./searchquery.html), GroupByQuery's [LimitSpec](./limitspec.html), and [BoundFilter](./filters.html#bound-filter).
+
+## Lexicographic
+Sorts values in case-sensitive lexicographic order.
```

Comment:
We can be more specific here. This sorts the UTF-8 representation of a string in lexicographic order, on a byte by byte basis.